zlib/conn.go

```diff
@@ -41,20 +41,20 @@ const (
 // Implements the net.Conn interface
 type Conn struct {
 	// Underlying network connection
-	conn    net.Conn
-	tlsConn *ztls.Conn
-	isTls   bool
+	conn                net.Conn
```

Comment:
Some auto-formatting from my IDE (IntelliJ w/ Go plugin). Let me know if we are using different auto-formatting standards.